src/snowflake/snowpark/dataframe.py

```diff
@@ -3550,7 +3550,9 @@ def dtypes(self) -> List[Tuple[str, str]]:
         return dtypes
 
     def _with_plan(self, plan):
```

Comment:
appreciate if you can add type hint for this method

Reply:
added